### ISSUE TITLE
Insufficient contrast - Items on Stage

### ISSUE THREAD

**Issue by daminihs** [wordplaydev/wordplay#721]
### What happened?

Element has insufficient color contrast of 2.49 (foreground color: #921a54, background color: #000000, font size: 10.0pt (13.3333px), font weight: normal). Expected contrast ratio of 4.5:1

### Anything else?

![Image](https://github.com/user-attachments/assets/ddab0972-2618-4697-bf29-d4bcba9faf81)

### What browsers are you seeing the problem on?

Chrome

### What operating system are you using?

Windows

**Comment by daminihs** [comment]
![Image](https://github.com/user-attachments/assets/7f736902-184e-43c4-94f4-3769966bc45c)
